config/mapconverter/overwritepropertiesmapconverter/properties_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package configmapprovider
+package overwritepropertiesmapconverter
```

Comment:
not sure about package name, seems a bit wordy... I'll admit that I don't have a better alternative for name though

Reply:
But it'll inconsistent with `expandmapconterter.New`, so I'd probably chose to keep `overwritepropertiesmapconverter` unless we can change `expandmapconterter` as well

Reply:
I would keep it as is, since we have a consistent way of naming.